test/e2e/far_e2e_test.go

```diff
@@ -0,0 +1,64 @@
+package e2e
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/medik8s/fence-agents-remediation/api/v1alpha1"
+)
+
+const fenceAgentDummy = "echo"
```

Comment:
(very) nit: I think the name of this variable could be more descriptive. Is it the Agent name?

Reply:
Nit: IMO var value could be better (i.e  for example dummy-fence-agent)

Reply:
`echo` is just an example to `dummy-fence-agent`  and I doubt would run without a failure. In the end a pod will try to run a cli command `dummy-fence-agent`, and I have used `echo` since it was most likely to be executed on any pod (and Linux based OS).